### DISCUSSION TITLE
请问怎么打开下载功能？

### DISCUSSION THREAD

**Discussion by harrison-ming** [Evil0ctal/Douyin_TikTok_Download_API#389] [Q&A] [answered]
我用Docker部署，用的默认配置文件。
请问怎么打开下载功能呢？

`{
    "code": 400,
    "message": "Download endpoint is disabled.",
    "support": "Please contact us on Github: https://github.com/Evil0ctal/Douyin_TikTok_Download_API",
    "time": "2024-05-11 06:35:09",
    "router": "/api/download",
    "params": {
        "url": "https://www.tiktok.com/@6ixbuzztv/video/7345107201373015302",
        "prefix": "true",
        "with_watermark": "false"
    }
}`

**Comment by Evil0ctal** [comment]
到容器里编辑config.yaml文件即可

**Comment by Evil0ctal** [comment]
并且默认的应该是开启状态，可以尝试手动重新拉取一下docker镜像。

**Comment by harrison-ming** [comment]
重新拉取docker镜像已解决，感谢帮助

**Reply by Evil0ctal** [reply]
不客气

**Comment by harrison-ming** [comment]
另外请教一下，crawler.py的接口怎么用，方便给一个示例吗？

**Reply by Evil0ctal** [reply, marked as answer]
查看endpoint文件夹中的代码内容
